torch_geometric/datasets/modelnet.py

```diff
@@ -51,23 +58,26 @@ class ModelNet(InMemoryDataset):
         '40': 'http://modelnet.cs.princeton.edu/ModelNet40.zip'
     }
 
-    def __init__(self, root, name='10', train=True, transform=None,
-                 pre_transform=None, pre_filter=None):
+    def __init__(self, root: str, name: Optional[str] = '10',
+                 train: Optional[bool] = True,
```

Comment:
The comments in the code state that train is optional?
 train (bool, optional): If :obj:`True`, loads the training dataset,
            otherwise the test dataset. (default: :obj:`True`)